alan_std.js

```diff
@@ -1083,15 +1085,15 @@ export async function runWindow(initialContextFn, contextFn, gpgpuShaderFn) {
     canvas.style['height'] = '100%';
     canvas.width = window.innerWidth;
     canvas.height = window.innerHeight;
-    document.body.addEventListener("resize", () => {
+    window.addEventListener("resize", () => {
       canvas.width = window.innerWidth;
       canvas.height = window.innerHeight;
     });
-    if (!context.cursorVisible) {
-      canvas.style['cursor'] = 'none';
-    }
     context.canvas = canvas;
   }
+  if (!context.cursorVisible) {
+    context.canvas.style['cursor'] = 'none';
+  }
```

Comment:
Moving this out of the other `if` block so it can be applied to a canvas provided by the user, not only one auto-generated by the code here.